### PR TITLE
Adding min_level to avoid log some messages unless specified

### LINKED ISSUE #4: Allow to filter minimum reporting level
we need to be able to avoid to log all messages (for example logging only panic, error and fatal logs)

### PR DESCRIPTION
**Pull request by saniales**
Fixes #4

### CODE REVIEW

**Review by shuLhan — APPROVED** [review]

**Comment by shuLhan** [comment]
Thank you.

File: hooks/logrus/mmhooklogrus.go [hunk 1]
```diff
@@ -60,7 +61,14 @@ type mmHookLogrus struct {
 //
 // [1] https://docs.mattermost.com/developer/message-attachments.html
 //
-func NewHook(endpoint, channel, username string, attc *Attachment) logrus.Hook {
+func NewHook(endpoint, channel, username string, attc *Attachment, minLevel logrus.Level) logrus.Hook {
+	levels := make([]logrus.Level, 0, len(logrus.AllLevels))
+	for _ lvl := range logrus.AllLevels {
```
Comment (saniales) [inline hooks/logrus/mmhooklogrus.go:66]: there is an error here, I corrected it but it does not show
opening pr again

**Comment by saniales** [comment]
Fixed by #7

**Comment by shuLhan** [comment]
Ok, #7 need to rebase to fix conflict.

**Pull request merged**